src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,4 @@
+%days-parameters {
+  width: $card-day-width;
+  height: $card-day-width;
+};
```

Comment:
I guess you could change it for the function and then you will get more flexibility